mod/groups/actions/groups/edit.php

```diff
@@ -210,21 +210,7 @@
 	));
 }
 
-$has_uploaded_icon = (!empty($_FILES['icon']['type']) && substr_count($_FILES['icon']['type'], 'image/'));
-
-if ($has_uploaded_icon) {
-	$filehandler = new ElggFile();
-	$filehandler->owner_guid = $group->owner_guid;
```

Comment:
ah ok... i did look if it was part of this PR, so it was probably taken care of earlier